make/prepare

```diff
@@ -80,6 +80,10 @@ def validate(conf, args):
         if storage_provider_config == "":
             raise Exception("Error: no provider configurations are provided for provider %s" % storage_provider_name)
 
+    redis_db_index = rcp.get("configuration", "redis_db_index").strip()
```

Comment:
we should add Redis host and port checking to make sure the redis config is conrrect because redis is a must-have component for js so far.